File: lesson3/python3/homework/homework3_10b.py
#!/usr/bin/python3
#: lesson3/python3/homework/homework3_10b.py
# д/з упражнение 10_b

x = float(input("Введите координата X => "))
y = float(input("Введите координата Y => "))

if x * x + y * y / 2 < 1 and x*x + y*y / 2 > -1:
    print("попали в круг")
else:
    print("не попали")

File: lesson3/c_c++/homework/homework3_10b.c
//: lesson3/c_c++/homework/homework3_10b.c
#include <stdio.h>

int main() {
    float x, y;
    printf("Введите первую кординату => ");
    scanf("%f", &x);
    printf("Введите вторую когдинату => ");
    scanf ("%f", &y);

    if (x*x + y*y / 2 < 1 && x*x + y*y / 2 > -1) {
        printf("попали.\n");
    } else {
        printf("непопали.\n");
    }
}
